index.html

```diff
@@ -6,5 +6,123 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <main>
+      <header>
+        <h1>Gesa Siebert</h1>
+        <nav>
+          <a href="" #biography>Biography</a>
+          <a href="#contact">Contact</a>
+        </nav>
+      </header>
+      <figure>
+        <img
+          src="images/profile-picture-gesa-siebert.jpg"
+          alt="Portrait of Gesa Siebert"
+        />
+        <figurecaption> This is me. </figurecaption>
+      </figure>
+      <section>
+        <h2>Things I'm interested in</h2>
+        <ul>
+          <li>Coding</li>
+          <li>Design</li>
+          <li>Nature</li>
+        </ul>
+        <h2>Things I like</h2>
+        <ul>
+          <li>white space</li>
+          <li>black cats</li>
+          <li>colors</li>
+        </ul>
+      </section>
+      <section id="#contact">
+        <div>Tel. 0179 – 12 34 56</div>
+        <div>Email: hallo@gesasiebert.de</div>
+      </section>
+      <section>
+        <figure>
+          <img ="images/github-brands.svg" alt="GitHub logo" />
+          <figcaption>GitHub</figcaption>
+        </figure>
+        <figure>
+          <img ="images/codepen-brands.svg" alt="codepen logo" />
+          <figcaption>codepen</figcaption>
+        </figure>
+        <figure>
+          <img ="images/linkedin-brands.svg" alt="linkedin logo" />
+          <figcaption>LinkedIn</figcaption>
+        </figure>
```

Comment:
You don't need a `figure` and `figcaption` tag everytime you use an icon.I would reccomend to put the images there by themselves. If you want them to link to your social media channels, you need to wrap them in `a` tags!

Reply:
Is the path to images correct?

How/where can I ask questions in my code within the pull request like the reviewers do?